morf-postgresql/src/main/java/org/alfasoftware/morf/jdbc/postgresql/PostgreSQLDialect.java

```diff
@@ -207,14 +207,22 @@ protected String getSqlFrom(Cast cast) {
     return ImmutableList.<String>builder()
         .addAll(preStatements)
         .add(createTableStatement.toString())
-        .add(addTableComment(table.getName()))
+        .add(addTableComment(table))
         .addAll(postStatements)
         .build();
   }
 
 
-  private String addTableComment(String tableName) {
-    return "COMMENT ON TABLE " + schemaNamePrefix() + tableName + " IS 'REALNAME:[" + tableName + "]'";
+  private String addTableComment(Table table) {
+    StringBuilder sqlBuilder = new StringBuilder("COMMENT ON TABLE ");
+    if (!table.isTemporary()) {
+      sqlBuilder.append(schemaNamePrefix());
+    }
+    return sqlBuilder.append(table.getName())
+      .append(" IS 'REALNAME:[")
+      .append(table.getName())
+      .append("]'")
+      .toString();

```

Comment:
Actually, there is org.alfasoftware.morf.jdbc.postgresql.PostgreSQLDialect.schemaNamePrefix(Table)
which already handles temp tables nicely. You sould simply use that here instead of schemaNamePrefix().

This should end up being a one-liner, as we want to move away from explicit StringBuilder usage
and let Java do the StringBuilder-behind-the-scenes optimisation.